src/pages/config/ConfigList.tsx

```diff
@@ -30,9 +24,9 @@ export function ConfigListPage() {
   });
 
   const search = params.get("search");
-  const tag = decodeURIComponent(params.get("tag") || "All");
-  const configType = decodeURIComponent(params.get("type") || "All");
-  const groupByProp = decodeURIComponent(params.get("groupByProp") ?? "");
+  const tag = params.get("tag") || "All";
```

Comment:
decodeURIComponent is needed as some of the values has special characters which gets encoded when we set query params. so when we read we need to decode.

Reply:
Ah, makes sense.